ISISComputingGroup/nicos-core
# Ticket 3478: Changed the way NICOS instrument is set

See https://github.com/ISISComputingGroup/IBEX/issues/3478

NICOS can use the instrument environment variable to say which instrument to start. This meant the nicos demo instrument was starting on NDXDEMO rather than IbexInstrument. Using this env varible simplifies our setup.

To test:
* Confirm NICOS runs on a dev machine
* Confirm NICOS runs ok on demo (these changes are currently on there)
* Confirm the INSTRUMENT environment variable is unchanged after starting NICOS
